### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by megan07**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,23 @@
 ## 3.16.0 (Unreleased)
+FEATURES:
+* **New Data Source:** monitoring: `google_monitoring_uptime_check_ips` ([#1912](https://github.com/terraform-providers/terraform-provider-google-beta/pull/1912))
```
Comment (danawillow) [inline CHANGELOG.md:3]:
```suggestion
* **New Data Source:** `google_monitoring_uptime_check_ips` ([#1912](https://github.com/terraform-providers/terraform-provider-google-beta/pull/1912))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,23 @@
...
+* **New Resource:** firebase: Added new `firebase_project_location` resource to finalize the firebase location. ([#1919](https://github.com/terraform-providers/terraform-provider-google-beta/pull/1919))
```
Comment (danawillow) [inline CHANGELOG.md:4]:
```suggestion
* **New Resource:** `firebase_project_location`: finalizes the firebase location. ([#1919](https://github.com/terraform-providers/terraform-provider-google-beta/pull/1919))
```

something like this? we generally just put the name of the new resource here, but I suppose the description can't hurt too

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,23 @@
...
+* cloudfunctions: added support for `vpc_connector_egress_settings` to `google_cloudfunctions_function` ([#1904](https://github.com/terraform-providers/terraform-provider-google-beta/pull/1904))
+* accesscontextmanager: added `status.vpc_accessible_services` to `google_access_context_manager_service_perimeter` to control which services are available from the perimeter's VPC networks to the restricted Google APIs IP address range. ([#1910](https://github.com/terraform-providers/terraform-provider-google-beta/pull/1910))
+* cloudrun: added ability to autogenerate revision name ([#1900](https://github.com/terraform-providers/terraform-provider-google-beta/pull/1900))
+* compute: added ability to resize `google_compute_reservation` ([#1908](https://github.com/terraform-providers/terraform-provider-google-beta/pull/1908))
+* container: added `enable_resource_consumption_metering` to `resource_usage_export_config` in `google_container_cluster` (beta only) ([#1901](https://github.com/terraform-providers/terraform-provider-google-beta/pull/1901))
```
Comment (danawillow) [inline CHANGELOG.md:12]:
```suggestion
* container: added `enable_resource_consumption_metering` to `resource_usage_export_config` in `google_container_cluster` ([#1901](https://github.com/terraform-providers/terraform-provider-google-beta/pull/1901))
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,4 +1,23 @@
...
+* container: added `resource_usage_export_config` to `google_container_cluster`, previously only available in `google-beta` (ga only) ([#1901](https://github.com/terraform-providers/terraform-provider-google-beta/pull/1901))
```
Comment (danawillow) [inline CHANGELOG.md:13]: this can be removed

File: CHANGELOG.md (hunk 1 above)
Comment (c2thorn) [inline CHANGELOG.md:3]:
```suggestion
* **New Data Source:** `google_monitoring_uptime_check_ips` ([#1912](https://github.com/terraform-providers/terraform-provider-google-beta/pull/1912))
```

**Pull request merged**